### DISCUSSION TITLE
nowy mega wątek `Grupa interia.pl`

### DISCUSSION THREAD

**Discussion by AroKol78** [MajkiIT/polish-ads-filter#20300] [Ideas] [answered]
propozycja

**Comment by xxcriticxx** [comment, marked as answer]
@AroKol78 test URL please

**Reply by AroKol78** [reply]
moje 13x zgłoszeń związanych/należących do interia/polsat (większość drobnica)
`https://github.com/MajkiIT/polish-ads-filter/issues?q=is%3Aissue+author%3AAroKol78`
plus zgłoszenia innych użytkowników

**Reply by xxcriticxx** [reply]
masz cos teraz co bys chcial zmienic? jesli tak to prosze o PR.

**Reply by AroKol78** [reply]
nowy mega wątek `Grupa interia.pl` tak jak `Grupa wp.pl`
------> tylko propozycja

**Reply by xxcriticxx** [reply]
ok tutaj czy na main?

**Reply by AroKol78** [reply]
> nowy mega wątek `Grupa interia.pl` tak jak `Grupa wp.pl`

> ok tutaj czy na main?

- jest `https://github.com/MajkiIT/polish-ads-filter/issues` --> `Grupa wp.pl`
- propozycja `https://github.com/MajkiIT/polish-ads-filter/issues` --> `Grupa interia.pl`

**Reply by xxcriticxx** [reply]
https://github.com/MajkiIT/polish-ads-filter/issues/20301
